scss/civicrm/common/_accordions.scss

```diff
@@ -5,42 +5,41 @@
 
   .crm-accordion-header {
```

Comment:
Is the `.crm-container` main wrapping selector (at the top of the file) needed at all? We are already doing some automatic wrapping in the gulpfile
```js
gulp.task('sass:civicrm', ['sass:sync'], function () {
  // ...

  .pipe(postcss([postcssPrefix({
    prefix: '.crm-container ',
    exclude: [/page-civicrm/, /crm-container/, /civicrm-menu/, /#root-menu-div/]
  }), postcssDiscardDuplicates]))
```
If it's not needed, please remove it